docs/installation/config/cmis.rst

```diff
@@ -99,6 +99,116 @@ Using the CMIS adapter
 
 .. _`CMIS adapter library`: https://github.com/open-zaak/cmis-adapter
 
+When using the CMIS-adapter, sometimes a Zaak, a Zaaktype or a Besluit need to be retrieved from the APIs.
+This happens for example when a Zaak folder is created, since the description of the Zaak is used as part of the folder name.
+
+For this reason, the CMIS-adapter needs to be given access to the various APIs.
+These are the steps to configure it:
+
+1. Go to **API Autorisaties > Applicaties** and click on **Applicatie toevoegen**.
+
+2. Enter a human readable label, for example ``CMIS adapter``.
+
+3. Enter a **Client ID**, for example ``cmis-adapter``.
+
+4. Enter a **Secret**. This should be a random string, for example ``Tl8@04&O4gXTtB``.
+
+5. Click **Opslaan en opnieuw bewerken**.
+
+6. Click **Beheer Autorisaties**.
+
+7. Under **Component** check ``Zaken API``, under **Selecteer scopes** check ``zaken.lezen``, under **Voor welke typen geldt dit?** check ``Alle huidige en toekomsitge ZAAKTYPEN`` and  under **Tot en met welke vertrouwelijkheidaanduiding?** check ``Zeer geheim``.
```

Comment:
Could you make this shorter, "Fill out the form:" like here: https://open-zaak.readthedocs.io/en/latest/installation/config/openzaak_config.html#open-zaak